app/views/layouts/_navbar.html.erb

```diff
@@ -1,7 +1,7 @@
 <nav class="navbar-container">
     <ul>
-        <li class="nav-item"> <%= image_tag  "navbar/bar-chart.svg" %> <%= link_to "Preview", '#'  %></li>
-        <li class="nav-item"><%= image_tag  "navbar/plus-circle.svg" %> <%= link_to "Create", '#' %></li>
-        <li class="nav-item"><%= image_tag  "navbar/upload.svg" %> <%= link_to "Upload", '#' %></li>
+        <li class="nav-item"> <%= image_tag  "navbar/bar-chart.svg" %> <%= link_to "Preview", visualization_index_path  %></li>
+        <li class="nav-item"><%= image_tag  "navbar/plus-circle.svg" %> <%= link_to "Create", visualization_create_path %></li>
```

Comment:
Also, we should be using the `visualization_new_path` here since `create` does not render anything but only submits the form.

Reply:
Good catch!